code/game/gamemodes/objective_items.dm

```diff
@@ -121,11 +121,21 @@
 	difficulty = 3
 	excludefromjob = list("Chief Engineer","Research Director","Station Engineer","Scientist","Atmospheric Technician")
 
-/datum/objective_item/steal/plasma/check_special_completion(obj/item/tank/T)
+/datum/objective_item/steal/plasma/check_completion() //So special that not even check_special_completion is good enough for it
 	var/target_amount = text2num(name)
 	var/found_amount = 0
-	found_amount += T.air_contents.gases[/datum/gas/plasma] ? T.air_contents.gases[/datum/gas/plasma][MOLES] : 0
-	return found_amount>=target_amount
+	for(var/datum/mind/M in owners)
+		if(!isliving(M.current))
```

Comment:
Check if that person is considered escaped out of custody instead

Reply:
No, you can complete the steal objective without completing your escape objective. If I changed it to that, then this objective would be the only steal objective that you fail if you don't escape.